apps/scheduler/internal/timers/expired_commands.go

```diff
@@ -0,0 +1,108 @@
+package timers
+
+import (
+	"context"
+	"log/slog"
+	"time"
+
+	"github.com/redis/go-redis/v9"
+	"github.com/samber/lo"
+	config "github.com/satont/twir/libs/config"
+	model "github.com/satont/twir/libs/gomodels"
+	"github.com/satont/twir/libs/logger"
+	commandscache "github.com/twirapp/twir/libs/cache/commands"
+	generic_cacher "github.com/twirapp/twir/libs/cache/generic-cacher"
+	"go.uber.org/fx"
+	"gorm.io/gorm"
+)
+
+type ExpiredCommandsOpts struct {
+	fx.In
+	Lc fx.Lifecycle
+
+	Logger logger.Logger
+	Config config.Config
+
+	Gorm        *gorm.DB
+	RedisClient *redis.Client
+}
+
+type expiredCommands struct {
+	config        config.Config
+	logger        logger.Logger
+	db            *gorm.DB
+	commandsCache *generic_cacher.GenericCacher[[]model.ChannelsCommands]
+}
+
+func NewExpiredCommands(opts ExpiredCommandsOpts) *expiredCommands {
+	timeTick := lo.If(opts.Config.AppEnv != "production", 15*time.Second).Else(5 * time.Minute)
```

Comment:
5 minutes is to high for this feature, i'd like to see 30secs - 1minute instead.